stew/shims/hashes.nim

```diff
@@ -0,0 +1,10 @@
+import
+  std/[hashes, typetraits]
+
+func hashAllFields*(x: object|tuple): Hash =
```

Comment:
this could simply be `hashFields` - `all` is kind of implicit (same as the `fields` magic iterator etc)